modules/apps/headless/headless-admin-user/headless-admin-user-impl/src/main/java/com/liferay/headless/admin/user/internal/jaxrs/exception/mapper/UserAccountEmailAddressMustNotBePOP3UserExceptionMapper.java

```diff
@@ -31,7 +32,10 @@ public class UserAccountEmailAddressMustNotBePOP3UserExceptionMapper
 	protected Problem getProblem(
 		UserEmailAddressException.MustNotBePOP3User mustNotBePOP3User) {
 
-		return new Problem(mustNotBePOP3User);
+		return new Problem(
+			Response.Status.BAD_REQUEST,
+			"The user account email address is invalid because it must not " +
```

Comment:
I feel this will not be merged

Reply:
I left only "is invalid" because I found another example where also if the Exception can have multiple reasons is always only "invalid"